lightning/src/ln/onion_utils.rs

```diff
@@ -110,24 +110,167 @@ pub(crate) fn next_hop_pubkey<T: secp256k1::Verification>(
 	curr_pubkey.mul_tweak(secp_ctx, &Scalar::from_be_bytes(blinding_factor).unwrap())
 }
 
-pub(super) trait HopInfo {
+trait HopInfo {
```

Comment:
I think this fixup 4911a3ab27536b24d1c62c79e2409daf1b4efe57 belongs to the commit `Trampoline payload construction method` instead